### DISCUSSION TITLE
Possible bug on select folder

### DISCUSSION THREAD

**Discussion by sergiohdljr** [usebruno/bruno#1557] [Q&A]
Today when i try to create a new folder and had to choose the path i had this bug. i don't know if it was on bruno or my OS, want to know if someone had the same problem before open a issue.

![Captura de tela de 2024-02-08 11-02-08](https://github.com/usebruno/bruno/assets/102623806/b5056b20-351c-48b4-9331-fb97c83ce58d)

**Comment by sanjai0py** [comment]
Possible duplicate of #1437
